src/Microsoft.ML.Core/Utilities/MathUtils.cs

```diff
@@ -2,11 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Float = System.Single;
-
 using System;
 using System.Collections.Generic;
-using System.Linq;
+using Float = System.Single;
```

Comment:
Should we rename `Float` in the file to `Single`? It seems odd having our own alias of `float`/`Single`.

Reply:
Seems we do this in many of the file... (not just this one file)

Reply:
May be we should but does it make sense to be it owns change so that its easier on the reviewers? Apparently there are 54 files where Float = System.Single is being used so I suspect removing them all and changing all their references to float might pollute this change. Lets see what @TomFinley feels? I'm fine either ways.

Reply:
It's unclear to me why we're bringing that up in this PR. There is no urgency to that change. We've been doing it opportunistically. Let's continue that policy. Doing it all at once would cause a great deal of conflicts with existing changes for no particular reason whatsoever. (These changes being at the *header* of the file are at least easier to manage.)

Reply:
Agreed. 

I initially didn't notice `using Float = System.Single;` was used in many files. If it was only in one, I'd ask for the change. It's a larger tasks which belongs in its own PR.